bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/forms2/contract/form/FormType.java

```diff
@@ -0,0 +1,16 @@
+package org.bahmni.module.bahmnicore.forms2.contract.form;
+
+public enum FormType {
+
+    ALL_OBSERVATION_TEMPLATE_FORMS("v1"), FORM_BUILDER_FORMS("v2");
+
+    private final String type;
+
+    FormType(String type) {
+        this.type = type;
+    }
+
+    public String get() {
```

Comment:
please avoid using such methods. In this case, overriden toString() would have been enough

Reply:
sure